src/scala/scripts/PBGenerateRequest.scala

```diff
@@ -10,13 +10,23 @@ object PBGenerateRequest {
     val jarOutput = args.get(0)
     val parsedProtoFiles = args.get(1).split(':').toList.map { rootAndFile =>
       val parsed = rootAndFile.split(',')
-      val root = parsed(0)
-      val file = if (root.isEmpty) {
-        parsed(1)
-      } else {
-        parsed(1).substring(root.length + 1)
+      if (parsed(1).contains("external")) {
```

Comment:
@ittaiz, I've found the bug manifested in OriP's branch. This line had the error.
changing it to `startsWith`:
```suggestion
      if (parsed(1).startsWith("external")) {
```

fixed it.

next step is to add E2E test.
should we do this in Wix's fork or on bazelbuild?